src/containers/course-sections-list/CourseSectionsList.tsx

```diff
@@ -146,11 +146,10 @@ const CourseSectionsList: FC<CourseSectionsListProps> = ({
     )
 
     return (
-      <>
+      <Fragment key={item.id}>
```

Comment:
don't understood. 
Why did you replace `<>` with just Fragment instead and used `key` value. Are you use some method for `CourseSectionsList` ?

Reply:
Thanks for question.  It was a tough one to fix and i struggled with it a lot. The reason I have added it is that `sectionItem` function is used inside of `map` in following code snippet:

```jsx
const sectionItems =
    items.length === 0
      ? clearCoorperationMenu
      : items.map((item) => sectionsItem(item))
```

When I was investigating error in console, it said that the error is present inside `<SortableContext items={items} strategy={verticalListSortingStrategy}>` in the following code:

```jsx
const courseSectionContent = enabled && (
    <>
      <SortableContext items={items} strategy={verticalListSortingStrategy}>
        {sectionItems}
      </SortableContext>
      <DragOverlay>{activeItem && sectionsItem(activeItem, true)}</DragOverlay>
    </>
  )
```

That's confused me, because `SortableContext` is nor being used inside any kind of loop. So I came up with solution that you can see now, because `cooperationMenu` and `SortableWrapper` are repeated on the same level and we should provide a wrapper for it to make an error disappear. I decided not no use extra `Box`, just sticked to `React`'s `Fragment`

I hope I have answered your question and it because clearer for you

Reply:
> Thanks for question. It was a tough one to fix and i struggled with it a lot. The reason I have added it is that `sectionItem` function is used inside of `map` in following code snippet:
> 
> ```js
> const sectionItems =
>     items.length === 0
>       ? clearCoorperationMenu
>       : items.map((item) => sectionsItem(item))
> ```
> 
> When I was investigating error in console, it said that the error is present inside `<SortableContext items={items} strategy={verticalListSortingStrategy}>` in the following code:
> 
> ```js
> const courseSectionContent = enabled && (
>     <>
>       <SortableContext items={items} strategy={verticalListSortingStrategy}>
>         {sectionItems}
>       </SortableContext>
>       <DragOverlay>{activeItem && sectionsItem(activeItem, true)}</DragOverlay>
>     </>
>   )
> ```
> 
> That confused me, because `SortableContext` is not being used inside any kind of loop. So I came up with solution that you can see now, because `cooperationMenu` and `SortableWrapper` are repeated on the same level and we should provide a wrapper for it to make an error disappear. I decided not no use extra `Box`, just sticked to `React`'s `Fragment`
> 
> I hope I have answered your question and it because clearer for you

I saw. Agree with you that `Box`, since a wrapper should't be here